src/test/java/io/jenkins/plugins/uleska/toolkit/ToolkitLocatorTest.java

```diff
@@ -0,0 +1,133 @@
+package io.jenkins.plugins.uleska.toolkit;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+
+public class ToolkitLocatorTest {
+
+    @InjectMocks
+    private ToolkitLocator toolkitLocator;
+
+    @Mock
+    private ToolkitApi toolkitApi;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    public void testFindToolkitByNameReturnsNothingWhenNameIsNull() {
+        //given
+        String name = null;
+
+        //when
+        Optional<Toolkit> maybeToolkit = toolkitLocator.findByName(name);
```

Comment:
Maybe not a big deal but why not just pass null here instead of creating a null variable?

Reply:
I'm happy with null here.